packages/extension/src/background/alarms.ts

```diff
@@ -23,3 +23,16 @@ export function setTransactionTrackerUpdateAlarm() {
     periodInMinutes: 0.5, // fetch transaction updates of existing transactions every half minute from onchain
   })
 }
+
+
+export const tokenListUpdateAlarm = "core:tokenList:update"
+export async function refreshTokenListUpdateAlarm() {
+  await browser.alarms.clear(tokenListUpdateAlarm)
+  setTokenListUpdateAlarm()
+}
+export function setTokenListUpdateAlarm() {
+  browser.alarms.create(tokenListUpdateAlarm, {
+    delayInMinutes: 0.02,
```

Comment:
In the `alarmInfo` section of the [document](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/alarms/create#parameters), it is mentioned:

```
If an extension tries to set delayInMinutes to a value < 1, 
or when to a value < 1 minute in the future, 
then the alarm will fire after 1 minute.
```

Does this mean that `delayInMinutes` cannot be less than 1 on Chrome?

Reply:
You are right. Thanks for the reference.  I will update the value and the comment.

Reply:
I also updated the `delayInMinutes` and `periodInMinutes` for `transactionTrackerUpdateAlarm`.